specification/security/resource-manager/Microsoft.Security/stable/2022-05-01/settings.json

```diff
@@ -0,0 +1,306 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+    "version": "2022-05-01"
```

Comment:
>2022-05-01" [](http://example.com/codeflow?start=16&length=11)

this is still incorrect , you need to add the previous version of settings.json as the first checkin or else we cannot do a diff to see the changes

Reply:
hi @raosuhas ,
please notice that as per @mentat9 request in the comments i started a new PR using OpenAPIHub with the previous version in a separate commit.
i added a link to the new PR in my previous comment ^ and i also pointed to the latest PR in my email to _ARM RP API Review_ (that you responded on)
in case you still missed it - here is the link again #19334